server/src/services/initialization/onInitialize.ts

```diff
@@ -119,14 +123,16 @@ function getSessionInfo(params: InitializeParams) {
 
   const workspaceFolders = params.workspaceFolders || [];
 
-  const clientName = params.clientInfo?.name;
+  const clientName: string | undefined = params.clientInfo?.name;
+  const clientVersion: string | undefined = params.clientInfo?.version;
```

Comment:
In reviewing I noticed that `clientInfo` has a version.
I pushed a commit that swaps out `vscodeVersion` and instead uses the `clientVersion` here.
To test I added both the clientName and clientVersion to the output logging on the lsp.